From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Patrick McCarty <patrick.mccarty@intel.com>
Date: Wed, 29 Sep 2021 16:01:30 -0700
Subject: [PATCH] tinygettext library should be static

This library is not installed, so it should be declared static.

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 lib/tinygettext/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/tinygettext/CMakeLists.txt b/lib/tinygettext/CMakeLists.txt
index 108339c..5a67ab4 100644
--- a/lib/tinygettext/CMakeLists.txt
+++ b/lib/tinygettext/CMakeLists.txt
@@ -94,7 +94,7 @@ file(GLOB TINYGETTEXT_HEADERS RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} include/tinyg
 
 ## define a target for building the library
 
-add_library(tinygettext ${TINYGETTEXT_SOURCES})
+add_library(tinygettext STATIC ${TINYGETTEXT_SOURCES})
 
 ## Add tinygettext dir to search path
 
